.github/workflows/prbuild.yml

```diff
@@ -97,4 +97,4 @@ jobs:
 
       - name: Tear down compose stack
         run: |
-          docker-compose -f docker-compose.yml -f docker-compose.ci.yml down
+          docker compose -f docker-compose.yml -f docker-compose.ci.yml down
```

Comment:
It could be that GH runner might able to recognise subcommand `compose` (as docker cli plugin) now. Last time I tried, it failed on me. But, worth give it a shot again. So, all good and try again now.

Reply:
Yes they are deprecating the "docker-compose" - and the GH runner now understands the newer compose. Is needed to get some of the newer compose functionality ("include")

Reply:
Right! Thanks for pointer.